spec/system/jobseekers/jobseekers_can_manage_a_profile_spec.rb

```diff
@@ -247,31 +243,18 @@
                                     ["Enter the date you started at this school or organisation", "#jobseekers-profile-employment-form-started-on-field-error"])
             end
           end
-
-          it "makes reason mandatory when non current" do
-            click_on I18n.t("buttons.save_and_continue")
-
-            expect(page).to have_css("ul.govuk-list.govuk-error-summary__list")
-
-            within "ul.govuk-list.govuk-error-summary__list" do
-              expect(page).to have_link("Enter your reason for leaving this role", href: "#jobseekers-profile-employment-form-reason-for-leaving-field-error")
-            end
-          end
         end
 
         it "associates an 'employment' with their jobseeker profile", :js do
           expect { add_jobseeker_profile_employment }.to change { profile.employments.count }.by(1)
         end
 
         context "when the form to add a new employment history entry is submitted" do
-          it "redirects to the review page" do
+          it "displays every employment history entry on the review page" do
```

Comment:
Combined these two tests as I don't think we needed one to just check the redirect on it's own, and combining them may slightly reduce run time.